mta/fixtures/stakeholder.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from conftest import generate_email, generate_name
+from swagger_client.models.api_stakeholder import ApiStakeholder
+
+
+@pytest.fixture()
```

Comment:
Please set the fixture's scope as `session`

Reply:
Done.